sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -337,8 +319,7 @@ def body_type(self):
         )
 
     @property
-    def properties(self):
-        # type: () -> Optional[AmqpMessageProperties]
+    def properties(self) -> Optional["AmqpMessageProperties"]:
```

Comment:
so for [Python 3.7+, we can actually add the following line](https://docs.python.org/3/whatsnew/3.7.html#pep-563-postponed-evaluation-of-annotations) at the top:
```
from __future__ import annotations
```
And you can then remove the forward reference here:
```suggestion
    def properties(self) -> Optional[AmqpMessageProperties]:
```